types/ember/test/ember-module-tests.ts

```diff
@@ -228,7 +228,7 @@ ma1.addObject('!'); // $ExpectType string
 ma1.filterBy(''); // $ExpectType NativeArray<string>
 // Ember.MutableEnumerable
 // tslint:disable-next-line:prefer-const
-let me1: Ember.MutableEnumerable<[string]> = null as any;
+let me1: Ember.MutableEnumerable<[string] | null | undefined> = null as any;
 me1.compact(); // $ExpectType NativeArray<[string]>
```

Comment:
This test was weird to begin with, IMHO, and could be improved to:
```suggestion
let me1: Ember.MutableEnumerable<string | null | undefined> = ['foo', null, undefined];
me1.compact(); // $ExpectType NativeArray<string>
```

Reply:
I'm also not sure why it uses `let` instead of `const`? Maybe to hint at mutability? IMHO it could be confusing since it's mutable regardless of `let` or `const`.

Reply:
Done.